hooks/boot/01-kafo-hook-extensions.rb

```diff
@@ -40,6 +40,10 @@ def local_redis?
     (foreman_server? && !param_value('foreman', 'jobs_sidekiq_redis_url')) || param_value('foreman_proxy::plugin::pulp', 'pulpcore_enabled')
   end
 
+  def needs_postgresql_scl_upgrade?
+    !File.exist?('/var/opt/rh/rh-postgresql12/lib/pgsql/data') && File.exist?('/var/lib/pgsql/data')
```

Comment:
@evgeni I had to add a condition here for this to work on fresh installations. Before adding the `&&` part, it attempted an upgrade on a fresh install.

Reply:
Makes sense.

This will still try to upgrade if the user installed postgres before running the installer, but I think that's fine as being labeled "unsupported"?